data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,462 @@
+			`	"Officially the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never did prove it had been conclusively terminated."`
+			`	"For the demonic symbol, we'll have to go to another library in room 4196A21."`
+			choice
+				`	(Take the elevator to the 419th floor.)`
+			`	When you arrive, Giti tells you, "This is a religious materials archive. Give me a minute here..." Her voice trails off as she starts typing on a terminal. She walks back and forth between several bookshelves skimming books for a few minutes.`
+			`	"I found it," she says. "You found a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. They were one of the first organizations to fund large-scale settlements on Freedom, though the symbol was ultimately co-opted by miscreants. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!"`
+			`	"The circle around the symbol looks almost like a wormhole. A red wormhole. I heard about one in Terminus near The Deep. Apparently nobody is able to enter one without their ship being torn to pieces. This symbol suggests there's another one in the middle of Syndicate space."`
+			`	"Fly through the wormhole at the coordinates on that card, if you can, come back and tell me what you see. In the meantime, I'll ask other researchers about the microtransceivers."`
+			choice
+				`	(Thank her and return to your ship.)`
+			`	Upon returning to your ship, you ponder Giti's findings. The coordinates are for a system in Syndicate space that is not reachable by starlane, so you'll need a Jump Drive. The red wormhole requires another key.`
+			`	Once you've passed through that wormhole, hopefully Giti will have more information for you.`
+				accept
+	on complete
+		log `Giti asked to go to the unexplored region beyond the Syndicate wormhole to seek a base. She's waiting at the Earth spaceport with 7 tons of cargo.`
+		conversation
+			`After a long series of shuttles, high-speed rails, and a several-minute-long elevator ride, you reach Giti's office.`
+			`	As soon as she sees you she asks, "What did you find out?" You explain your journey to her.`
+			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. They should not have been so visible, but they were due to some strange anomalies. The strangest of all was not a star; it was an object with unknown properties that confuse our scanners even today."`
```

Comment:
`They should not have been so visible, but they were due to some strange anomalies`
This sentence, especially the second use of `they`, is a little confusing. Can you think of another way to phrase it? I'd suggest something myself, but I think I might butcher its meaning.

Reply:
```suggestion
			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. By all rights these stars should not have been so visible, but some strange anomaly made that part of the sky clearer for a while. The strangest thing we spotted was not a star; it was some kind of object with unknown properties that confused our scanners. Even today I'm not sure we'd make sense of it, but the anomaly passed and we couldn't even make an attempt now."`
```
Thoughts?

Reply:
Just one comma missing (added below). But yeah, that helps a whole lot
```suggestion
			`	She responds, "Wait... I... lemme check..." followed by some typing on a nearby terminal. "Yes, you're right. In the late 21st century, there was an effort to map stars in that intergalactic region. By all rights these stars should not have been so visible, but some strange anomaly made that part of the sky clearer for a while. The strangest thing we spotted was not a star; it was some kind of object with unknown properties that confused our scanners. Even today I'm not sure we'd make sense of it, but the anomaly passed, and we couldn't even make an attempt now."`
```